UPGRADE.md

```diff
@@ -2,6 +2,18 @@
 
 ## dev-develop
 
+### Smart-Content
+
+The smart-content is now able to deep-link selected elements. To provide proper-links add the deep-links to the
+DataProvider configuration.
+
+```php
+$this->configuration = Builder::create()
+    ...
+    ->setDeepLink('content/contents/{webspace}/{locale}/edit:{id}/details')
+    ->getConfiguration();
+```
```

Comment:
I think we've agreed on not putting new features into the `UPGRADE.md`? Probably this PR is older than this decision :smiley: